.travis.yml

```diff
@@ -58,7 +64,12 @@ script:
 #  - python ../tests/test_pyccel_openmp.py --openmp --execute
 #  - python ../tests/test_pyccel_lapack.py --libs='blas lapack' --execute
 #  - python ../tests/test_pyccel_mpi.py --compiler=mpif90
+  - coverage combine
+  - coverage xml
+  - rm ${SITE_DIR}/pyccel_cov.pth
   - cd -
 
 after_script:
   - rm -rf __pytest__
+  - export CODACY_PROJECT_TOKEN=d5e147769dad48158549d7236cb5d53a
```

Comment:
This one should not be written directly, but set as a protected environment variable in Travis CI. See https://travis-ci.org/pyccel/fffi/settings for an example.

Reply:
@yguclu 
I suspect I don't have the rights to do this. When I try to set the variable I get a message:
`Oh no! There was an error saving this environment variable.`
If I try again it tells me "Variable with this name is already defined."
But, given the output from travis I am sure that the variable isn't defined

Reply:
Now that it's in the Git history anyway, doesn't matter ;-) but for the future this would be the clean way. In my opinion we can leave it in.

Reply:
In that case this change either needs to be rolled back or Yaman needs to add the environment variable before carrying out the merge

Reply:
@krystophny We should probably have asked you before doing this ;-) The Codacy documentation that we read did not mention Travis unfortunately. I suppose I can revoke that token and create a new one?

Reply:
Now this is what we should have read: https://support.codacy.com/hc/en-us/articles/207279819-Coverage

Reply:
I have revoked the previous Codacy API token, created a new one, and uploaded it to Travis.

Reply:
That’s good, this way we don’t have to rewrite history (else this should be done before merging to master)